src/lib.rs

```diff
@@ -327,29 +322,6 @@ pub enum MouseButton {
     Button8                = ffi::MOUSE_BUTTON_8,
 }
 
-// We can't use `enum_from_primitive!` for MouseButton due to
-// https://github.com/andersk/enum_primitive-rs/issues/2
-// We implement FromPrimitive manually instead.
-impl num::FromPrimitive for MouseButton {
```

Comment:
Why not keep the `from_i64` and `from_u64` as non-trait methods?

Reply:
After thinking about this, I decided I would prefer to replace them (and the other FromPrimitive implementations) with a `from_i32` that does a range check and a transmute.